src/cancel-reminders/lambda/models.ts

```diff
@@ -0,0 +1,25 @@
+import {
+	createDetailedValidator,
+	registerType,
+} from 'typecheck.macro/dist/typecheck.macro';
+import { isValidEmail } from '../../lib/models';
+
+export interface Cancellation {
+	identity_id: string;
+}
+
+type Email = string;
```

Comment:
I initially tried sharing this type with the `create-reminders-lambda`, but that actually doesn't work. Becuase macros process a file at a time, `typecheck.macro` needs all of the types in one file, else it won't actually validate it

Reply:
oh that's annoying!